backend/src/ee/routes/v1/ssh-certificate-authority-router.ts

```diff
@@ -0,0 +1,257 @@
+import { z } from "zod";
+
+import { EventType } from "@app/ee/services/audit-log/audit-log-types";
+import { sanitizedSshCa } from "@app/ee/services/ssh/ssh-certificate-authority-schema";
+import { SshCaStatus } from "@app/ee/services/ssh/ssh-certificate-authority-types";
+import { sanitizedSshCertificateTemplate } from "@app/ee/services/ssh-certificate-template/ssh-certificate-template-schema";
+import { SSH_CERTIFICATE_AUTHORITIES } from "@app/lib/api-docs";
+import { readLimit, writeLimit } from "@app/server/config/rateLimiter";
+import { verifyAuth } from "@app/server/plugins/auth/verify-auth";
+import { AuthMode } from "@app/services/auth/auth-type";
+import { CertKeyAlgorithm } from "@app/services/certificate/certificate-types";
+
+export const registerSshCaRouter = async (server: FastifyZodProvider) => {
```

Comment:
should we expose a public endpoint for fetching the public key of a CA? 

vault has something like this which makes setting it up for the host machine easier
`curl -o /etc/ssh/trusted-user-ca-keys.pem http://127.0.0.1:8200/v1/ssh-client-signer/public_key`

<img width="726" alt="image" src="https://github.com/user-attachments/assets/1155634e-2b42-4088-8b82-b2c6ded8822d">

Reply:
Note: Will add new (unauthenticated) dedicated endpoint to get public key of CA back